.github/workflows/publish-schemata.yml

```diff
@@ -40,6 +40,9 @@ jobs:
 
       - name: Commit changes
         run: |
+          git config --global user.email "test@github.land"
+          git config --global user.name "GitHub Almighty"
```

Comment:
The commit will be pushed using dandibot user token, so let's make it here a bit more specific like
```
          git config --global user.email "team@dandiarchive.org"
          git config --global user.name "DANDI Bot"
```

Reply:
Done.